packages/devtools_app/lib/src/memory/memory_allocation_table_view.dart

```diff
@@ -91,6 +114,10 @@ class AllocationTableViewState extends State<AllocationTableView>
       setState(() {});
     });
 
+    addAutoDisposeListener(controller.monitorAllocationsNotifier, () {
+      setState(() {});
```

Comment:
you don't need to add this setState. If you don't specify a listener, setState is the default for addAutoDisposeListener

Reply:
Done.